packages/at_client/lib/src/service/notification_service.dart

```diff
@@ -103,6 +103,28 @@ abstract class NotificationService {
   Future<NotificationResult> getStatus(String notificationId);
 
   /// Returns the [AtNotification] of the given notificationId
+  ///
+  /// Usage:
+  ///
+  ///  To fetch a notification with id: abc-123
+  /// ```dart
+  /// var notification = NotificationServiceImpl(atClient!);
```

Comment:
This isn't how we create a NotificationServiceImpl right now; we expose NotificationServiceImpl.create(...)

Reply:
You could just write `NotificationService notifSvc = <code to get handle to notification service>` so that the docs stay current

Reply:
@gkc: Updated the dart docs as mentioned above.